androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/ManageDataLauncherActivity.java

```diff
@@ -294,4 +295,21 @@ public void onCustomTabsServiceConnected(ComponentName componentName,
         @Override
         public void onServiceDisconnected(ComponentName componentName) {}
     }
+
+    public static boolean packageSupportsSiteSettings(String packageName, PackageManager packageManager) {
+        if (packageName != null) {
+            if (ChromeLegacyUtils.supportsTrustedWebActivities(packageManager, packageName)) {
+                // Chrome 72-74 support site settings but don't yet have the category
```

Comment:
This comment implies that Chrome 75 has the category *for Site Settings*. (I'm guessing you copied this from the other code where it was referring to the category for TWAs in general.)